index.js

```diff
@@ -1,8 +1,9 @@
 import Prism from 'prismjs';
+const md = require('markdown-it')();
```

Comment:
We don’t need to import `markdown-it` because we’ll get an instance on plugin initialisation. Please delete this line and see below for a better way.